src/types/PoolKey.sol

```diff
@@ -3,6 +3,9 @@ pragma solidity ^0.8.0;
 
 import {Currency} from "./Currency.sol";
 import {IHooks} from "../interfaces/IHooks.sol";
+import {PoolIdLibrary} from "./PoolId.sol";
+
+using PoolIdLibrary for PoolKey global;
```

Comment:
can you then remove `using PoolIdLibrary for PoolKey` frm all the contracts where it is used?

Reply:
Please can you also sign your commit :)